src/index.ts

```diff
@@ -24,19 +25,23 @@ function onError(error: Record<string, unknown>) {
   // handle specific listen errors with friendly messages
   switch (error.code) {
   case "EACCES":
-    console.error(bind + " requires elevated privileges");
+    logger.logError(bind + " requires elevated privileges", error);
+
```

Comment:
I think we should also include a logger.logeError on  line 21

Reply:
I agree!!!